deps/src/ideals.cpp

```diff
@@ -235,8 +235,8 @@ void singular_define_ideals(jlcxx::Module & Singular)
                     ring im) {
 
         rChangeCurrRing(pr);
-        nMapFunc nMap =n_SetMap(currRing->cf, im->cf);
-        return maMapIdeal(map_id, pr, im_id, im, nMap);
+        //nMapFunc nMap =n_SetMap(currRing->cf, im->cf);
```

Comment:
I would be inclined to remove lines, rather than comment them. We can always see what they were by looking back in Git history.